website/www/site/content/en/get-started/downloads.md

```diff
@@ -87,6 +87,13 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.23.0 (2020-07-16)
+Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.23.0/apache-beam-2.23.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.23.0/apache-beam-2.23.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.23.0/apache-beam-2.23.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12347145).
```

Comment:
Please add a link to the new blog post here. (I realize this introduces a minor circular dependency between the website update and blog post PRs; perhaps we could combine these into one PR, since they're meant to be merged simultaneously anyway?)

Reply:
Done.  I already merged the blogpost PR but yes, it actually would make sense to combine the changes in one PR. FYI: @youngoli

Reply:
I updated and re-assigned BEAM-10487.